dev-qt/kdsingleapplication/kdsingleapplication-1.0.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="KDSingleApplication is a helper class for single-instance policy applications"
+HOMEPAGE="https://github.com/KDAB/KDSingleApplication/"
+SRC_URI="https://github.com/KDAB/KDSingleApplication/releases/download/${PV}/${P}.tar.gz"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="qt6 static"
```

Comment:
Do we even need `qt6` flag if the only consumer yet is only requiring it with Qt6?

Reply:
yes, upstream supports qt5 and qt6 builds, and it should be honored here. hardcoding towards qt6 doesn't really help us.

Reply:
if there are no revdeps, why would we care troubling users with it?

Reply:
> hardcoding towards qt6 doesn't really help us.

The correct question is: Does providing qt5 help us in any way? If the answer is " ", there is no reason to have a use flag.